octoprint_mrbeam/iobeam/laserhead_handler.py

```diff
@@ -50,14 +52,29 @@ def __init__(self, plugin):
 
     @property
     def _current_used_lh_model_string(self):
-        if str(self._current_used_lh_model_id) in self._LASERHEAD_MODEL_STRING_MAP:
-            return self._LASERHEAD_MODEL_STRING_MAP.get(str(self._current_used_lh_model_id))
-        else:
-            raise ValueError("Unknown laserhead model ID {!r}".format(self._current_used_lh_model_id))
-
-
+        """
+        Returns the laserhead model name
+
+        Returns:
+            str: laserhead model or None if Model name is not found
+        """
+        self._logger.debug("Current laserhead ID: {}, Current LaserHead Model: {}".format(
+            self._current_used_lh_model_id, self._current_used_lh_model))
+        laser_head_model_str_list = [v for (k, v) in self._LASERHEAD_MODEL_STRING_MAP.items()
+                                     if ((v == str(self._current_used_lh_model)) or
+                                         (k == str(self._current_used_lh_model_id)))]
+        try:
+            return laser_head_model_str_list.pop(0)
+        except IndexError:
+            self._logger.error("Unknown laserhead model,  name: {} ID: {}".format(
+                self._current_used_lh_model, self._current_used_lh_model_id))
+            return str(None)
```

Comment:
Why don't you want to raise the Exception? I thought "None"s were bad to have in our values

Reply:
in this case we don't need to crash the system because the laserhead string is none, because after iobeam is up. it will have a valid string value, so we log the error and we handle it